app/controllers/users_controller.rb

```diff
@@ -1,69 +1,73 @@
 class UsersController < ApplicationController
-  before_action :set_user, only: %i[show edit update destroy]
+  before_action :authenticate_user!
+  # before_action :set_user, only: %i[show edit update destroy]
 
   # GET /users or /users.json
   def index
     @users = User.all
   end
 
   # GET /users/1 or /users/1.json
-  def show; end
-
-  # GET /users/new
-  def new
-    @user = User.new
+  def show
+    @user = User.find(params[:id])
+    @foods = @user.foods.all
   end
 
-  # GET /users/1/edit
-  def edit; end
+  # # GET /users/new
```

Comment:
[OPTIONAL]
- I think this comment block is too large to leave. I suggest deleting lines 16-72